src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -22,31 +22,38 @@
 
 //	Bookmarks of places to mine. Add additional bookmarks separated by comma.
 string[] SetMiningSiteBookmark = new[] {
-	"mining_site_bookmark_name",
+	"Asteroid Belts",
```

Comment:
yeah you rigth but in my case i only need astroid belts you can put what you want here